README.md

```diff
@@ -1,58 +1,56 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
```

Comment:
For this PR we should assume all work is being in Docker whether on Linux or OS X (this would change for OS X in #307). Can we remove these requirements here, except for docker and docker-compose?